### DISCUSSION TITLE
Notebook Updated

### DISCUSSION THREAD

**Discussion by tiprock-network** [tiprock-network/MedicineDemand-TimeSeries#3] [Announcements]
Hi everyone the notebook has been updated with an appropriate model with RMSE of about 3, kindly @aaryan2134 check for the possible errors and make an issue so we can resolve in your own branch. Other members are as well welcomed to check the model out and give suggestions.

**Comment by aaryan2134** [comment]
Yes. I will check it out and update accordingly

**Reply by tiprock-network** [reply]
Okay thank you
